docs/security/10-01-troubleshooting.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Basic troubleshooting
+type: Troubleshooting
+---
+
+## 403 Forbidden in the Console
+
+If you log to the Console and get 403 Forbidden error, do the following:
+
+  1. Fetch the ID Token. For example, use the [Chrome Developer Tools](https://developers.google.com/web/tools/chrome-devtools) and search for the token in sent requests.
+  2. Decode the ID Token. For example, use the [jwt.io](https://jwt.io/) page.
+  3. Check if the token contains groups claims:
+      ```
+      "groups": [
+        "example-group"
+      ]
+      ```
+  4. Make sure the group you are assigned to has [permissions](https://kyma-project.io/docs/components/security/#details-roles-in-kyma) to view resources you requested.
+    
+## Problems with certificates on Gardener
+
+During installation on Gardener Kyma requests domain SSL certificates using Gardener's custom resource [`Certificate`](https://gardener.cloud/050-tutorials/content/howto/x509_certificates/#request-a-certificate-via-certificate) to grant a secure communication using both Kyma UI and Kubernetes CLI. If:
+ - `xip-patch` installation takes too long or an error occurs `Certificate is still not ready, status is {STATUS}. Exiting...`,
+ - you notice any issues regarding certificates validity,
+  
+  follow the steps:
+
+1. Check the status of the Certificate resource. Run:
+
+    ```kubectl get certificates.cert.gardener.cloud --all-namespaces```
+
+2. If status of any Certificate is `Error`, run:
+
+    ```kubectl get certificates -n {CERTIFICATE_NAMESPACE} {CERTIFICATE_NAME} -o jsonpath='{ .status.message }'```
+
+The result describes the reason for the failure of issuing a domain SSL certificate. You can create a new Certificate resource applying suggestions from the error message to request a new domain SSL certificate. Follow these steps:
+    
+1. Make sure the secret connected to the Certificate resource is not present on the cluster. To find its name and namespace, run:
+
+    ```kubectl get certificates -n {CERTIFICATE_NAMESPACE} {CERTIFICATE_NAME} -o jsonpath='{ .spec.secretRef }'```
+    
+2. Delete the incorrect Certificate from the cluster.
+
```

Comment:
Plase mention examples for apiserver-proxy cert job as well. 
When it comes to cert errors, there might be a situation of exceeded 64 characters length (provisioner will ensure that) however, someone from open source using Gardener API can bypass that. In such case applying cert with fixed domain length will not help cause xip-patch sets different domain in the override based on what is in the shoot cm